.github/workflows/auto-issue-update.yml

```
@@ -0,0 +1,61 @@
+  sync-issues:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged != false || github.event.action == 'opened' || github.event.action == 'edited'
+
+    permissions:
+      issues: write
+      pull-requests: read
+
+    steps:
+      - name: 🧠 Extract Issue Numbers from PR Description
+        id: extract
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)